spanner_orm/field.py

```diff
@@ -171,17 +184,16 @@ def validate_type(value: Any) -> None:
   def matches(type: str) -> bool:
     return type == 'FLOAT64'
 
-  @staticmethod
-  def support_length() -> bool:
-    return False
-
 
 class String(FieldType):
   """Represents a string type."""
 
   @staticmethod
-  def ddl() -> str:
-    return 'STRING(MAX)'
+  def ddl(length: Optional[int] = None) -> str:
+    len = 'MAX'
+    if length:
+      len = str(length)
+    return f'STRING({len})'
```

Comment:
(nit) This is confusing having two variables with almost the same name. Also, `len` shaddows a builtin function. How about `length_ddl` instead of `len`?